particles/Processing/PresentLabel.particle

```diff
@@ -0,0 +1,7 @@
+import './schemas/Inference.schema'
+import './schemas/MlResource.schema'
+
+particle PresentLabel in './js/PresentLabel.js'
+  in [Classification] predictions
+  consume root
+  //description `show the top k labels (and confidence values)`
```

Comment:
Is there a convention for writing documentation for particles, e.g. an equivalent to a javadoc? 

If so, I think we can re-use these descriptions as documentation if we don't want them here.

Reply:
There is no convention for that, we can consider it.

Agreed, documentation value is why I didn't erase them completely.